projectlist.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
I noticed you are using material Icons and in `index.html` they are using font awesome 
https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/blob/c9533c33c8c5f99b9a580bd3d7dc73f71faeb1ac/index.html#L11-L14

@eddiejaoude Is it fine to use multiple icon libraries ?

Reply:
Great spot @alrifay 👍  . You are right, we do not need another icon library included. Font-awesome has enough icons